Some minor edits (#4)
greenelab/iscb-diversity-manuscript: content/10.methods.md

We gleaned the full name of the fellow as well as the year in which they received the honor.
We used the name as provided on the site. For certain methods we were required to split the full name into first and last names.
In this case we chose the first non-initial name as the first name and the final name as the last name.

> We did not consider a hyphen to be a name separator, so both side of hyphenated names were included.

Reviewer: ```suggestion
We did not consider a hyphen to be a name separator: for hyphenated names, all components were included.
```
"sides" feels like an odd way to refer to this...  but I'm not sure this is better 🤷‍♂

Author: How about "parts" or "components"? 
> We did not consider a hyphen to be a name separator: for hyphenated names, all components were included.

Reviewer: 👍

#### RECOMB Keynote Speakers

We examined the webpage for each RECOMB meeting.

> We were able to successfully find pages with keynote speakers for the years 1997-2019.

Reviewer: I kept the one with gaps because the webpages do not have the additional years. The following sentence clarifies where the additional information came from.
```suggestion
We were able to successfully find conference webpages with keynote speakers for 1999, 2000, 2001, 2004, 2007, 2008, and 2010-2019.

Author: Ah sorry, I missed the following sentence! I was editing yesterday's version. Should we clarify these as "webpages" or "conference webpages"?

Author: Also, I realized the 2016 proceedings have 2 tables with all speaker names from 1997-2016. Maybe we don't need to cite the other ones.

Reviewer: I went with what was in the final column of the spreadsheet. You did the curation so I trust your judgement. I would reference what you used, just in case there are differences. I also think "conference webpages" is better than webpages, so 👍 on that change.

Author: When I found the second table in the 2016 proceedings, I went through all the years and double-checked all the names from 1997-2016! So there were not be any differences. I will comment on the spreadsheet for a couple of instances where talks were canceled due to emergency reasons but I still think we should include those names.

We predicted the race and ethnicity of honorees and authors using the R package `wru`.
`wru` implements methods described in Imai and Khanna [@doi:10.1093/pan/mpw001] to predict race and ethnicity using surname and location information.
The underlying data used for prediction are derived from the United States Census.
We used only the surname of author or honoree to make predictions via the `predict_race()` function.

> We slightly modify the source code to enable the function to return inconclusive results and prevent its automatic imputation of average demographic distribution for names absent from the census.

Reviewer: ```suggestion
However, in the case of names that were not observed in the census, the function's behavior was to use the average demographic distribution from the census.
We modified the function to return a status denoting that results were inconclusive instead.
```

Author: Thank you for suggesting this. I didn't quite know how to phrase this. How about a shorter version:

> We modified the method's source code to enable the function's return of inconclusive results because the unmodified version performed undesirable imputation using the average demographic distribution for names absent from the census.

Reviewer: What about breaking it into two sentences?
> However, in the case of names that were not observed in the census, the function's behavior was to use the average demographic distribution from the census.
> We modified the function to return a status denoting that results were inconclusive instead.

Reviewer: I updated my suggestion to this one ☝️. Feel free to make your own changes though and merge when you are comfortable with it.

Author: I like it!
